Userland/Libraries/LibTest/TestCase.h

```diff
@@ -8,16 +8,36 @@
 #pragma once
```

Comment:
General feedback on c31af3245cd9b63381391e6de0804ca6627fb69e: This commit seems to have ended up with random changes that don't match the description and the `RANDOMIZED_TEST_CASE` macro that _is_ mentioned in the description isn't added here.

Reply:
Thanks, I'll investigate. It's possible I've messed something up in my fixups when rebasing for the other review rounds